### ISSUE TITLE
Link to examples is hard to follow

### ISSUE THREAD

**Issue by ArtskydJ** [schemacrawler/SchemaCrawler#1524]
### Description

In https://www.schemacrawler.com/schemacrawler-grep.html I clicked the "SchemaCrawler examples" link which brought me to [github.com/schemacrawler/SchemaCrawler/releases/](https://github.com/schemacrawler/SchemaCrawler/releases/).  I assumed it was an incorrect link as I didn't see any links for downloading examples.  I downloaded the Source Code zip, and didn't see an "examples" folder.  After a while, I did find it in the distribution file.

Maybe there's a way to make it more obvious?
- Mention the "distribution" file?
- Have a download like "examples.zip"?

A second issue is to get the <1 MB of examples, I had to download >380 MB zip file.

---

Anyway, I wanted to bring to your attention that the messaging could be clearer.  Feel free to close this issue.

**Comment by ArtskydJ** [comment]
A second disappointment is that the grep example in the zip file is not that different from what's in https://www.schemacrawler.com/schemacrawler-grep.html and is actually less advanced.

<details>
<summary>screenshots</summary>

## website:

![image](https://github.com/schemacrawler/SchemaCrawler/assets/1833684/f6eb1d4f-7148-4e04-a0f8-bf11992e425a)

## from distribution.zip examples/grep/grep-readme.html

![image](https://github.com/schemacrawler/SchemaCrawler/assets/1833684/cf75d776-3f83-48dd-9cdd-bfb2e700924e)

</details>

**Comment by sualeh** [comment]
@ArtskydJ Thanks for reporting this. I will fix the documentation.

**Comment by sualeh** [comment]
@ArtskydJ I have addressed this.
